src/library/entities/Library.java

```diff
@@ -84,153 +86,155 @@ public static synchronized void SaVe() {
     }
 
     
-    private long gEt_NeXt_ItEm_Id() {
-        return NeXt_ItEm_Id++;
+    private long getNextItemId() {
+        return nextItemId++;
     }
 
     
-    private long gEt_NeXt_PaTrOn_Id() {
-        return NeXt_PaTrOn_Id++;
+    private long getNextPatronId() {
+        return nextPatronId++;
     }
 
     
-    private long gEt_NeXt_LoAn_Id() {
-        return NeXt_lOaN_Id++;
+    private long getNextLoanId() {
+        return nextLoanId++;
     }
 
     
-    public List<Patron> lIsT_PaTrOnS() {        
-        return new ArrayList<Patron>(PaTrOnS.values()); 
+    public List<Patron> listPatrons() {
+        return new ArrayList<Patron>(patrons.values());
     }
 
 
-    public List<Item> lIsT_ItEmS() {        
-        return new ArrayList<Item>(CaTaLoG.values()); 
+    public List<Item> listItems() {
+        return new ArrayList<Item>(catalog.values());
     }
 
 
-    public List<Loan> lISt_CuRrEnT_LoAnS() {
-        return new ArrayList<Loan>(CuRrEnT_LoAnS.values());
+    public List<Loan> listCurrentLoans() {
+        return new ArrayList<Loan>(currentLoans.values());
     }
 
 
-    public Patron aDd_PaTrOn(String firstName, String lastName, String email, long phoneNo) {        
-        Patron PaTrOn = new Patron(firstName, lastName, email, phoneNo, gEt_NeXt_PaTrOn_Id());
-        PaTrOnS.put(PaTrOn.GeT_ID(), PaTrOn);        
-        return PaTrOn;
+    public Patron addPatron(String firstName, String lastName, String email, long phoneNo) {
+        Patron patron = new Patron(firstName, lastName, email, phoneNo, getNextPatronId());
+        patrons.put(patron.GeT_ID(), patron);
+        return patron;
     }
 
     
-    public Item aDd_ItEm(String a, String t, String c, ItemType i) {        
-        Item ItEm = new Item(a, t, c, i, gEt_NeXt_ItEm_Id());
-        CaTaLoG.put(ItEm.GeTiD(), ItEm);        
-        return ItEm;
+    public Item addItem(String a, String t, String c, ItemType i) {

```

Comment:
Parameter names need to be meaningful and conform to class diagram:

To:

```
String author, String title, String callNumber, ItemType itemType
```

Reply:
PR1-renamed parameter names for addItem method to conform with CSG1&2 @haynescrossley  @elliotmitchum 
commit q7914ada43ac3c323d0f07daab095f8eb4bf54e71